libwebrtc/src/native/audio_source.rs

```diff
@@ -134,6 +134,7 @@ impl NativeAudioSource {
 
         let mut inner = self.inner.lock().await;
         let mut samples = 0;
+        let enable_queue  = self.sys_handle.audio_options().enable_queue;
```

Comment:
I believe this always returns `false`, because the `enable_queue` option is not initialized explicitly in the C++ implementation of this `audio_options` method.

Does the `enable_queue` bit even need to be passed into C++? Why not just store the `enable_queue` as a field on the rust `NativeAudioSource`?

Reply:
hey, yes, there is a bug here, I created another PR to fix it
https://github.com/livekit/rust-sdks/pull/360

Reply:
After modification, `enable_queue` is an optional parameter of `NewAudioSourceRequest` instead of `AudioSourceOptions`.